helm/charts/hydra/Chart.yaml

```diff
@@ -3,7 +3,7 @@ appVersion: "v2.0.1"
 description: A Helm chart for deploying ORY Hydra in Kubernetes
 name: hydra
 icon: https://raw.githubusercontent.com/ory/docs/master/docs/static/img/logo-hydra.svg
-version: 0.26.1
+version: 0.26.2
```

Comment:
Please do not edit that, as it is auto-generated :)

Reply:
Done